README.md

```diff
@@ -9,6 +9,56 @@ Version 2.0. See the file "[LICENSE](LICENSE)" for more information.
 
 This is the Orders storage module.
 
+## PREPARATION
+
+### Tenant Initialization
+
+A tenant can be initialized in the following ways and a parameter can be passed to control the Sample data load
```

Comment:
Maybe start by stating that sample data is optionally loaded during tenant initialization... then continue with what you have here...

Reply:
Done